cpp/test/neighbors/ann_cagra.cuh

```diff
@@ -987,11 +994,12 @@ inline std::vector<AnnCagraInputs> generate_inputs()
     }
   }
 
-  // refinement options
+  // Refinement options
+  // Varying host_dataset, ivf_pq_search_refine_ratio
   inputs2 =
     raft::util::itertools::product<AnnCagraInputs>({100},
                                                    {5000},
-                                                   {32, 64},
+                                                   {64},
```

Comment:
I'm not sure about this one either. Without this, we'd only be checking refinement in a single index configuration (host/device dataset + several refinement variants).

Reply:
Happy to revert.
```suggestion
                                                   {32, 64},
```